spec/models/in_person_enrollment_spec.rb

```diff
@@ -412,9 +411,50 @@
       freeze_time do
         enrollment = create(
           :in_person_enrollment,
-          enrollment_established_at: days_ago_established_at.days.ago,
+          enrollment_established_at: (validity_in_days - 3).days.ago,
         )
-        expect(enrollment.days_to_due_date).to eq(validity_in_days - days_ago_established_at)
+        expect(enrollment.days_to_due_date).to eq(3)
+      end
+    end
+
+    context 'check edges to confirm date calculation is correct' do
+      it 'returns the correct due date and days to due date with 1 day left' do
+        freeze_time do
+          enrollment = create(
+            :in_person_enrollment,
+            enrollment_established_at: (validity_in_days - 1).days.ago,
+          )
+          expect(enrollment.days_to_due_date).to eq(1)
+          expect(enrollment.due_date).to(
+            eq(Time.zone.now + 1.day),
+          )
+        end
+      end
+
+      it 'returns the correct due date and days to due date with 0.5 days left' do
+        freeze_time do
+          enrollment = create(
+            :in_person_enrollment,
+            enrollment_established_at: (validity_in_days - 0.5).days.ago,
+          )
+          expect(enrollment.days_to_due_date).to eq(0)
```

Comment:
@aduth great callout. I _think_ we're ok. I'm currently digging through the `ready_to_verify_presenter_spec`: I just added two tests locally for 1 day remaining and 0 days remaining. 1 day remaining works, 0 days remaining fails because we get a -1, as expected by your observation.

The other test already in that file passes, so I think the behavior is expected. We do have a `in_person_deadline_passed` email that goes out, and I'm just digging around now for the logic that controls when that is sent so I can confirm that on 0 days remaining that email is sent and the `in_person_ready_to_verify_reminder` email is not. I'll confirm in another message once I'm done digging.

Reply:
Ok 👍 Thanks for checking